src/domain/eventGroups/detail/__tests__/EventGroupsDetailActions.test.jsx

```diff
@@ -55,11 +55,11 @@ describe('<EventGroupsDetailActions />', () => {
     });
     getWrapper();
 
-    await waitFor(() => {
-      expect(
-        screen.getByRole('link', { name: 'eventGroups.actions.addEvent.do' })
-      ).toBeInTheDocument();
-    });
+    expect(
+      await screen.findByRole('link', {
+        name: 'eventGroups.actions.addEvent.do',
+      })
+    ).toBeInTheDocument();
```

Comment:
findByRole already asserts presence: The findByRole query will automatically throw an error if it cannot find the element within a timeout period. Therefore, explicitly asserting toBeInTheDocument() afterwards is unnecessary. If expect-toBeInTheDocument is wanted to be used, it should be used with a synchronous getter and then, if it wants to be waited still, waitFor is needed.

Reply:
Fair point, in this test case this is the only `expect` clause so removing that doesn't make much sense. So, not taking this code into account.